core/src/main/scala/java/text/SimpleDateFormat.scala

```diff
@@ -10,6 +10,67 @@ class SimpleDateFormat(private[this] val pattern: String,
 
   //def this() = this("???", DecimalFormatSymbols.getInstance())
 
+  def format(date: Date): String = {
```

Comment:
I think an obvious issue is that this doesn't seem to be localized. It should probably delegate to `symbols` to format some of the output